admin/services/check-form.service.js

```diff
@@ -332,6 +332,23 @@ const checkFormService = {
 
   removeWindowAssignment: async (formId, windowId) => {
     return checkFormDataService.sqlRemoveWindowAssignment(formId, windowId)
+  },
+
+  getParsedCheckForms: async (batchIds) => {
```

Comment:
I suggest just calling the arg: `ids` - it doesn't know anything about the caller's context, and doesn't need to know.

Reply:
@jon-shipley done